ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/textinput/v1/textinput/README.md

```diff
@@ -54,6 +54,7 @@ BLOCK cmp-adaptiveform-textinput
     ELEMENT cmp-adaptiveform-textinput__label
     ELEMENT cmp-adaptiveform-textinput__label-container
     ELEMENT cmp-adaptiveform-textinput__widget
+    ELEMENT cmp-adaptiveform-textinput__charcount
```

Comment:
you will need to wrap cmp-adaptiveform-textinput__maxchar inside it.

Reply:
Done